### PR TITLE
Update readme with listened and generated events

### PR DESCRIPTION
**Pull request by macartur**
Fix #587

### CODE REVIEW

**Comment by macartur** [comment]
Rebased and updated.

File: README.rst [hunk 1]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+Events
+######
+
+******
+Listen
+******
+
+kytos/core.openflow.raw.in
+==========================
+  Handle a RawEvent and generate a kytos/core.messages.in.* events.
```
Comment (cemsbr) [inline README.rst]: event.

File: README.rst [hunk 2]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+
+Content
+=======
```
Comment (cemsbr) [inline README.rst]: I believe this should be one level below "kytos/core.openflow.raw.in" (like @diraol did in of_topology).

File: README.rst [hunk 3]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+
+.. code-block:: python3
+
+    { 'message': <object> # instance of napps.kytos.of_core.utils.GenericHello message
+      'source': <object>, # instance of kytos.core.switch.Connection class
+    }
+
+EventContent
+============
```
Comment (cemsbr) [inline README.rst]: Not sure what EventContent is doing here.

File: README.rst [hunk 4]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+
+kytos/of_core.v0x01.messages.in.ofpt_stats_reply
+================================================
+  Listen to any input of OpenFlow StatsReply in versions 1.0 (v0x01) and
+  updates the switches list with its Flow Stats.
+
+Content
+=======
```
Comment (cemsbr) [inline README.rst]: Fix hierarchy level.

File: README.rst [hunk 5]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+    }
+
+
+kytos/of_core.v0x0[14].messages.in.ofpt_features_reply
+======================================================
+  Listen to any input of OpenFlow FeaturesReply in versions 1.0 (v0x01) or 1.3
+  (v0x04).
+
+Content
+=======
```
Comment (cemsbr) [inline README.rst]: Same thing

File: README.rst [hunk 6]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+      'source': <object>, # instance of kytos.core.switch.Connection class
+    }
+
+kytos/of_core.v0x04.messages.in.ofpt_multipart_reply
+====================================================
+  Listen to any input of OpenFlow MultiPartReply in versions 1.3 (v0x04) and
+  handles Port Description Reply messages
+
+Content
+=======
```
Comment (cemsbr) [inline README.rst]: Idem

File: README.rst [hunk 7]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+      'source': <object>, # instance of kytos.core.switch.Connection class
+    }
+
+kytos/of_core.v0x0[14].messages.in.ofpt_echo_request
+====================================================
+  Listen to any input of OpenFlow EchoRequest in versions 1.0 (v0x01) or
+  1.3 (v0x04) and generate an appropriate echo reply.
+
+Content
+=======
```
Comment (cemsbr) [inline README.rst]: Idem

File: README.rst [hunk 8]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+    }
+
+
+kytos/of_core.v0x0[14].messages.out.ofpt_echo_reply
+===================================================
+  Listen to any output of OpenFlow EchoReply in versions 1.0 (v0x01) or
+  1.3 (v0x04).
+
+Content
+=======
```
Comment (cemsbr) [inline README.rst]: Idem

File: README.rst [hunk 9]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+      'destination': <object>, # instance of kytos.core.switch.Connection class
+    }
+
+kytos/of_core.v0x0[14].messages.out.ofpt_features_request
+=========================================================
+  Listen to any output of OpenFlow FeaturesRequest in versions 1.0 (v0x01) or
+  1.3 (v0x04) and ensure request has actually been sent before changing state.
+
+Content
+=======
```
Comment (cemsbr) [inline README.rst]: Idem (omitting comments for the same error from now on).

File: README.rst [hunk 10]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+********
+Generate
+********
+
+kytos/of_core.hello_failed
+==========================
+Send Error message and emit event upon negotiation failure.The content of an
+event will have the following format:
+
+.. code-block:: python3
```
Comment (cemsbr) [inline README.rst]: Content section is missing

File: README.rst [hunk 11]
```diff
@@ -27,6 +24,280 @@ repository. To install this NApp, run:
...
+
+    { 'message': <object>, # instance of a python-openflow StatsRequest message
+      'destination': <object> # instance of kytos.core.switch.Connection class
+    }
+
+kytos/of_core.v0x01.messages.out.ofpt_echo_request
+==================================================
+Send an EchoRequest to a datapath.
+
+.. code-block:: python3
```
Comment (cemsbr) [inline README.rst]: Content section is missing (omitting comments for the same error below)

**Comment by beraldoleal** [comment]
@diraol @cemsbr can you please, make sure that your changes are ok and approve ?

**Pull request merged**